insights/parsers/jboss_version.py

```diff
@@ -1,45 +1,60 @@
 """
-JBoss version - File ``$JBOSS_HOME/version.txt``
-================================================
-
-This module provides plugins access to file ``$JBOSS_HOME/version.txt``
-
-Typical content of file ``$JBOSS_HOME/version.txt`` is::
-
-    Red Hat JBoss Enterprise Application Platform - Version 6.4.3.GA
-
-This module parses the file content and stores data in the dict ``self.parsed``.
-The version info can also be got via ``obj.major`` and ``obj.minor``, etc.
-
-Examples:
-    >>> jboss_version.file_path
-    '/home/test/jboss/jboss-eap-6.4/version.txt'
-    >>> jboss_version.raw
-    'Red Hat JBoss Enterprise Application Platform - Version 6.4.3.GA'
-    >>> jboss_version.major
-    6
-    >>> jboss_version.minor
-    4
-    >>> jboss_version.release
-    3
-    >>> jboss_version.version
-    '6.4.3'
-    >>> jboss_version.code_name
-    'GA'
+JBoss version
+=============
+Provide information about the versions of all running Jboss on a system.
 """
 from .. import Parser, parser
 from ..specs import Specs
+import json
+from insights.tests import context_wrap
 
 
 @parser(Specs.jboss_version)
 class JbossVersion(Parser):
-    """Parses the content of file ``$JBOSS_HOME/version.txt``."""
+    """
+    JBoss version - File ``$JBOSS_HOME/version.txt``
+    ================================================
```

Comment:
It's not a good idea to add such a doc title here.

Reply:
Remove them.